template.html

```diff
@@ -0,0 +1,143 @@
+<div id='success' style='display:none;'>
```

Comment:
Since you have three states related to the task I think it will be more clear if you prefix your ids by "task_" : task_success, here.